contexts/SettingsContext.tsx

```diff
@@ -50,6 +51,11 @@ const initialSettings = {
     [NetworkType.TestnetT10]: "wss://ws.tn10.kaspa.forbole.com/borsh",
     [NetworkType.TestnetT11]: "wss://ws.tn11.kaspa.forbole.com/borsh",
   },
+  kasplexApiUrls: {
+    [NetworkType.Mainnet]: "https://api.kasplex.org/v1",
+    [NetworkType.TestnetT10]: "https://tn10api.kasplex.org/v1",
+    [NetworkType.TestnetT11]: "https://tn11api.kasplex.org/v1",
+  },
```

Comment:
Can you add migration script for it?

Reply:
f439b313bf062ced1112bb02aa7627bb3e2257a1